client/src/scenes/books/Books.scss

```diff
@@ -1,3 +1,22 @@
 .books {
   padding: 1rem 1.185rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .book-list {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    row-gap: 14px;
+    column-gap: 43px;
```

Comment:
In the future, we will follow the style of this site https://shopify-demo.gatsbyjs.com/. For now, I ignore commenting on this part.